tools/protoc-gen-jekyll/testdata/test1.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+// Floating comment
+
+// Package-level comment
+package testpkg;
+
+import "google/protobuf/duration.proto";
+import "testdata/test2.proto";
+import "testdata/test3.proto";
+import "options.proto";
+
+option (istio.docs.front_matter_title) = "Hello";
+option (istio.docs.front_matter_overview) = "There";
+option (istio.docs.front_matter_layout) = "docs";
+option (istio.docs.front_matter_location) = "/testpkg.html";
```

Comment:
I am not super happy about proto annotations used solely for docs.
Every consumer of this proto now needs to import doc options, which is entirely orthogonal to most use cases.

Reply:
It's a fairly modest tax.

It would also be possible to embed this info as specially formatted comments.

Reply:
I vote for specially formatted (jekyll-style?) comment. The mandatory import of annotations feels clunky in proto design IMHO.

Reply:
Special formatting is preferred over annotation.

Reply:
Done. You now use instead:

// $title: My Title
// $overview: My Overview
// $location: https://mysite.com/mypage.html